### ISSUE TITLE
Add half-dvd option to todisc

### ISSUE THREAD

**Issue by grepper** [grepper/tovid#68]
_From [vmol...@gmail.com](https://code.google.com/u/109160704348794787408/) on July 24, 2008 17:42:08_

In the "wouldn't it be nice" category: allow the -half-dvd option that
tovid takes. A quick glance at the todisc code makes me think that it would
need to be added to the idvid program as well, although maybe there's a
work around.

Thanks,
Steve

_Original issue: http://code.google.com/p/tovid/issues/detail?id=67_

**Comment by grepper** [comment]
_From [grepper@gmail.com](https://code.google.com/u/grepper@gmail.com/) on July 29, 2008 03:51:46_

Does your DVD player object to a full sized menu and half-dvd videos ?  I have a
feeling this one is in the 'not part of the specs but will work in most DVD players'
category.  I really haven't investigated this at all.

grepper

**Status:** Questions

**Comment by grepper** [comment]
_From [vmol...@gmail.com](https://code.google.com/u/109160704348794787408/) on July 29, 2008 20:45:24_

No, it's not an issue with the DVD, it's simply that the 'todisc' program, which
provides exactly the tool I want, doesn't recognize the '-half-dvd' option. It would
be a straight pass-through to tovid, but might require more hacking in idvid. Right
now, the '-dvd' and '-vcd' options are just handed directly to tovid and idvid, but
that won't work (AFAIK, just from reading manpages, not code) with the -half-dvd.

(This should be marked as an 'enhancement', not 'defect', but I don't seem to have
access to those settings.)

**Comment by grepper** [comment]
_From [grepper@gmail.com](https://code.google.com/u/grepper@gmail.com/) on July 30, 2008 04:57:46_

in todisc 0.31 options are passed to tovid with -tovidopts (-tovidopts "-half-dvd
-ffmpeg -etc").  In current svn all tovid options are honoured directly on the todisc
command line, though '-noask' is passed with '-noprompt' to avoid conflict with the
todisc option of the same name.

These should be detailed in the man pages for both 0.31 and SVN todisc.

grepper

**Comment by grepper** [comment]
_From [vmol...@gmail.com](https://code.google.com/u/109160704348794787408/) on July 31, 2008 13:34:46_

Ah, sorry. I did see the "-tovidopts" option in the man page, but assumed (yeah, I
know) that handing todisc both "-dvd" (which todisc does by default) and "-half-dvd"
would mess things up. Now I've tried it, and it works fine, at least as far as
creating the image and playback in vlc, so if my DVD player chokes on it, that's my
problem.

Apparently I can't close this bug, but feel free, of course.

Steve
